include/mruby/error.h

```diff
@@ -17,11 +17,14 @@ MRB_BEGIN_DECL
 struct RException {
   MRB_OBJECT_HEADER;
   struct iv_tbl *iv;
-  struct RString *mesg;
+#ifdef MRB_WORD_BOXING
```

Comment:
I think this is unnecessary complexity. Should we always use `struct RObject* mesg`?

Reply:
I understand. Then let's do so.